src/sentry/south_migrations/0476_incidents.py

```diff
@@ -0,0 +1,1409 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = True
+
+    def forwards(self, orm):
+        # Adding model 'Incident'
+        db.create_table('sentry_incident', (
+            ('id', self.gf('sentry.db.models.fields.bounded.BoundedBigAutoField')(primary_key=True)),
+            ('organization', self.gf('sentry.db.models.fields.foreignkey.FlexibleForeignKey')(
```

Comment:
@mattrobenolt Could you bless this? I'm adding a foreign key to organization, which has worked ok before.

SQL:

```
 CREATE TABLE "sentry_incident" (
   "id" bigserial NOT NULL PRIMARY KEY,
   "organization_id" bigint NOT NULL,
   "identifier" integer NOT NULL,
   "detection_uuid" uuid NOT NULL,
   "status" smallint CHECK ("status" >= 0) NOT NULL,
   "title" text NOT NULL, "query" text NOT NULL,
   "date_started" timestamp with time zone NOT NULL,
   "date_detected" timestamp with time zone NOT NULL,
   "date_added" timestamp with time zone NOT NULL,
   "date_closed" timestamp with time zone NOT NULL
)
ALTER TABLE "sentry_incident"
  ADD CONSTRAINT "sentry_incident_organization_id_56facaa712033034_uniq" UNIQUE ("organization_id", "identifier")
CREATE TABLE "sentry_incidentproject" (
  "id" bigserial NOT NULL PRIMARY KEY,
  "project_id" integer CHECK ("project_id" >= 0) NOT NULL,
  "incident_id" bigint NOT NULL
)
ALTER TABLE "sentry_incidentproject"
  ADD CONSTRAINT "sentry_incidentproject_project_id_6bdb72a1d37644fb_uniq" UNIQUE ("project_id", "incident_id")
ALTER TABLE "sentry_incident"
  ADD CONSTRAINT "organization_id_refs_id_e3e51435" FOREIGN KEY ("organization_id") REFERENCES "sentry_organization" ("id") DEFERRABLE INITIALLY DEFERRED;
CREATE INDEX "sentry_incident_organization_id" ON "sentry_incident" ("organization_id");
ALTER TABLE "sentry_incidentproject"
  ADD CONSTRAINT "incident_id_refs_id_c8831f42" FOREIGN KEY ("incident_id") REFERENCES "sentry_incident" ("id") DEFERRABLE INITIALLY DEFERRED;
CREATE INDEX "sentry_incidentproject_incident_id" ON "sentry_incidentproject" ("incident_id");
```

Reply:
Yeah, this is fine, I also don't think we need to set `is_dangerous` flag here unless you're nervous about it.

Reply:
Not overly, I'll make it not dangerous.